draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1332,25 +1324,46 @@ Internet-Draft                 ACTN POI                    January 2023
 
 4.1. Optical Topology Discovery
 
-   The WSON Topology Model or, alternatively, the Flexi-grid Topology
-   model is used to report the DWDM network topology (e.g., ROADM nodes
-   and links), depending on whether the DWDM optical network is based on
-   fixed grid or flexible-grid.
+   The WSON Topology Model and the Flexi-grid Topology model can be used
+   to report the DWDM network topology (e.g., ROADM nodes and links),
+   depending on whether the DWDM optical network is based on fixed-grid
+   or flexible-grid or a mix of fixed-grid and flexible-grid.
 
-   The OTN Topology Model is used to report the OTN network topology
-   (e.g., OTN switching nodes and links), when the OTN switching layer
-   is deployed within the optical domain.
+   It is worth noting that, as described in xxx, a fixed-grid can also
```

Comment:
Appendix I of [ITU-T_G.694.1]

Reply:
Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90